src/WireFormat.h

```diff
@@ -148,6 +154,17 @@ enum ControlOp {
 };
 
 /**
+ * Used in linearizable RPCs to check whether or not the RPC can be processed.
```

Comment:
This says how the class is used, but not what it is. Can you also add a sentence for that?